pkg/globalconfig/global_config.go

```diff
@@ -537,25 +537,33 @@ func WriteProjectList(projects map[string]*ProjectInfo) error {
 	return nil
 }
 
-// GetGlobalDdevDir returns ~/.ddev, the global caching directory
+// GetGlobalDdevDir returns the global configuration directory, which varies by OS.
 func GetGlobalDdevDir() string {
+	// Check for legacy directory.
 	userHome, err := os.UserHomeDir()
 	if err != nil {
 		logrus.Fatal("Could not get home directory for current user. Is it set?")
 	}
 	ddevDir := filepath.Join(userHome, ".ddev")
 
-	// Create the directory if it is not already present.
+	// If legacy directory doesn't exist, create new one.
```

Comment:
This is creating the .ddev directory, nothing legacy about it. It may be trying to create the parent as a result also...

Reply:
The language here is ambiguous, sorry. I'll clarify in the comment.